app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/presenter/AuthenticationPresenterTest.kt

```diff
@@ -0,0 +1,41 @@
+package com.github.HumanLearning2021.HumanLearningApp.presenter
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.firebase.ui.auth.AuthUI
+import com.github.HumanLearning2021.HumanLearningApp.model.DummyDatabaseService
+import com.github.HumanLearning2021.HumanLearningApp.model.User
+import com.google.firebase.auth.ktx.auth
+import com.google.firebase.ktx.Firebase
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.tasks.await
+import org.hamcrest.Matchers.*
+import org.junit.Assert.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class AuthenticationPresenterTest {
+    private lateinit var presenter: AuthenticationPresenter
+
+    @Before
+    fun setUp() {
+        presenter = AuthenticationPresenter(AuthUI.getInstance(), DummyDatabaseService())
+    }
+    @Test
+    fun test_currentUser() {
+        assertThat(presenter.currentUser, nullValue())
+    }
+
+    @Test
+    fun test_signInAnonimously() = runBlocking {
```

Comment:
```suggestion
    fun test_signInAnonymously() = runBlocking {
```

will fix on the next push